sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsClientTestBase.java

```diff
@@ -268,6 +296,57 @@ void recognizeEntitiesTooManyDocumentsRunner(
             Arrays.asList(documentInput, documentInput, documentInput, documentInput, documentInput, documentInput));
     }
 
+    // Personally Identifiable Information Entity runner
+    void recognizePiiSingleDocumentRunner(Consumer<String> testRunner) {
+        testRunner.accept(PII_ENTITY_INPUTS.get(0));
+    }
+
+    void recognizePiiLanguageHintRunner(BiConsumer<List<String>, String> testRunner) {
+        testRunner.accept(PII_ENTITY_INPUTS, "en");
+    }
+
+    void recognizeBatchPiiEntityDuplicateIdRunner(Consumer<List<TextDocumentInput>> testRunner) {
+        testRunner.accept(getDuplicateTextDocumentInputs());
+    }
+
+    void recognizePiiEntitiesLanguageHintRunner(BiConsumer<List<String>, String> testRunner) {
+        testRunner.accept(PII_ENTITY_INPUTS, "en");
+    }
+
+    void recognizeBatchPiiEntitySingleErrorRunner(Consumer<List<TextDocumentInput>> testRunner) {
+        List<TextDocumentInput> inputs = Collections.singletonList(new TextDocumentInput("2", " "));
+        testRunner.accept(inputs);
+    }
+
+    void recognizeBatchPiiEntitiesRunner(Consumer<List<TextDocumentInput>> testRunner) {
+        testRunner.accept(TestUtils.getTextDocumentInputs(PII_ENTITY_INPUTS));
+    }
+
+    void recognizeBatchPiiEntitiesShowStatsRunner(
+        BiConsumer<List<TextDocumentInput>, TextAnalyticsRequestOptions> testRunner) {
+        final List<TextDocumentInput> textDocumentInputs = TestUtils.getTextDocumentInputs(PII_ENTITY_INPUTS);
+        TextAnalyticsRequestOptions options = new TextAnalyticsRequestOptions().setIncludeStatistics(true);
+
+        testRunner.accept(textDocumentInputs, options);
+    }
+
+    void recognizeStringBatchPiiEntitiesShowStatsRunner(
+        BiConsumer<List<String>, TextAnalyticsRequestOptions> testRunner) {
+        testRunner.accept(PII_ENTITY_INPUTS, new TextAnalyticsRequestOptions().setIncludeStatistics(true));
+    }
+
+    void recognizePiiEntitiesTooManyDocumentsRunner(Consumer<List<String>> testRunner) {
+        final String documentInput = PII_ENTITY_INPUTS.get(0);
+        // max num of document size is 5
+        testRunner.accept(
+            Arrays.asList(documentInput, documentInput, documentInput, documentInput, documentInput, documentInput));
+    }
+
+//    void recognizePiiEntitiesWithDomainRunner(BiConsumer<List<String>, TextAnalyticsRequestOptions> testRunner) {
```

Comment:
Does this still need to be commented, could you add a corresponding TODO here?

Reply:
this should be removed as well. We can add it back later when domain feature is on. Forget to delete it.